nemo/collections/llm/modelopt/quantization/quantizer.py

```diff
@@ -232,7 +270,7 @@ def quantize(self, model: "MegatronParallel", forward_loop=None):
                 unwrapped_model, "*input_quantizer", lambda amax: torch.clamp(amax, min=0.01 * maxbound)
             )
 
-        if parallel_state.get_tensor_model_parallel_rank() == 0:
+        if is_global_rank_zero():
             mtq.print_quant_summary(unwrapped_model)
```

Comment:
If we only print on global rank 0 then for PP ranks > 1, we won't see a quantizer summary. Maybe that's what we want - but just pointing out